codespell_lib/data/dictionary.txt

```diff
@@ -32690,6 +32694,8 @@ laguagues->languages
 laguange->language
 laguanges->languages
 laiter->later
+lamba->lambda, lambs,
+lambas->lambdas, lambs,
```

Comment:
Also `lambast`/`lambaste`, as well as `lamas`, `mambas`?

Reply:
I'll add `lambast`. I couldn't find the other typos.